### PR TITLE
fix context route to context root #2490

### PR DESCRIPTION
**Pull request by bttamuratsh**
Please review #2490.

### CODE REVIEW

File: source_en/ArchitectureInDetail/WebServiceDetail/SOAP.rst [hunk 1]
```diff
@@ -394,7 +394,7 @@ Application configuration
     * - Sr. No.
       - Description
     * - | (1)
-      - | Add \ ``[server projectName]-ws.xml``\  to reading target while generating a route \ ``ApplicationContext``\ .
+      - | Add \ ``[server projectName]-ws.xml``\  to reading target while generating a root \ ``ApplicationContext``\ .
  
 |
```
Comment (btyoshikawaa) [inline source_en/ArchitectureInDetail/WebServiceDetail/SOAP.rst]: - [ ] Is it unnecessary to correspond to these?
* https://github.com/terasolunaorg/guideline/blob/81d780b4f5088e271d4abe54024521193013f2da/source_en/ArchitectureInDetail/WebServiceDetail/REST.rst#L2171
* https://github.com/terasolunaorg/guideline/blob/81d780b4f5088e271d4abe54024521193013f2da/source_en/ArchitectureInDetail/WebServiceDetail/REST.rst#L2197

**Comment by btyoshikawaa** [comment]
@bttamuratsh 
And please re-base!

File: source_en/ArchitectureInDetail/WebServiceDetail/SOAP.rst (hunk 1 above)
Comment (bttamuratsh) [inline source_en/ArchitectureInDetail/WebServiceDetail/SOAP.rst]: @btyoshikawaa 
I also fixed wrong word in REST.rst and re-based.

Comment (bttamuratsh) [inline source_en/ArchitectureInDetail/WebServiceDetail/SOAP.rst]: @btyoshikawaa 
I also fixed wrong words in REST.rst and re-based.

Comment (bttamuratsh) [inline source_en/ArchitectureInDetail/WebServiceDetail/SOAP.rst]: @btyoshikawaa 
I also fixed wrong words in REST.rst and re-based.

**Comment by btyoshikawaa** [comment]
@bttamuratsh 
Please back-port!

**Pull request merged**